CONTRIBUTING.md

```diff
@@ -20,13 +20,13 @@
    make install
    ```
 
-1. Install the stdlib of scripts/nodes:
+1. Install the Griptape Nodes Library:
 
    ```shell
    ./install.sh
    ```
 
-   If you have Griptape Nodes API key, pass it as an argument to the install script:
+   If you plan on using [https://nodes.griptape.ai/](https://nodes.griptape.ai/), pass your API key as an argument to the install script:
```

Comment:
Much better!

Can we include clear instructions on how the user should be getting their API key?